### ISSUE TITLE
bug: Suspected PowerShell & Console Window Host processes which leak memory and cause high CPU usage on Windows

### ISSUE THREAD

**Issue by niutech** [janhq/jan#1178]
**Describe the bug**
When I run Jan 0.4.3 on Windows, even without any models loaded, it takes 4GB of RAM and 50% of CPU. In the task manager there are new `powershell.exe` and `conhost.exe` processes being created constantly, which cause a memory leak and high CPU usage, which looks very suspected. What are they doing and why?

**To Reproduce**
1. Open Jan on Windows
2. Open task manager and observe processes

**Expected behavior**
Jan should be lightweight and don't create any extra processes.

**Screenshots**
![Screenshot](https://i.postimg.cc/t4hxqPP2/Jan-0-4-3.png)

**Desktop (please complete the following information):**
 - OS: Windows 11

**Comment by hiento09** [comment]
The issue arises from our monitoring extension, which employs the `systeminformation` library (https://www.npmjs.com/package/systeminformation) within Node. This library, when operating on Windows, uses PowerShell to execute commands that gather system information. In the upcoming Pull Request, I intend to streamline the process by excluding unnecessary details. This will help in reducing both the RAM usage and the quantity of consoles produced.

**Comment by hiento09** [comment]
will be resolve from version 0.4.4

**Comment by niutech** [comment]
This is mad that a single npm package which does nothing important but getting device info takes half of my CPU and leaks memory by running hundreds of processes. You shouldn't include it without testing above all. Why didn't you use a node built-in functions: `os.freemem()` and `os.cpus()` or a tiny wrapper on top of it: [os-utils.getCPUUsage()](https://github.com/oscmejia/os-utils/blob/master/lib/osutils.js#L165)?

**Comment by hiento09** [comment]
Thank you for bringing this to our attention. I understand your concerns about the performance issues and the memory leak caused by the npm package we used for retrieving device information. Your suggestion to utilize node's built-in functions like `os.freemem()` and `os.cpus()`, or even a lightweight wrapper like `os-utils.getCPUUsage()`, is indeed a more efficient approach. We prioritized certain features in the package that seemed beneficial at the time, but it seems we overlooked its impact on performance. We will revisit our choice and conduct thorough testing to ensure better efficiency and memory management in our future updates. Your feedback is invaluable, and we're committed to improving our application's performance.
